src/main/java/racingcar/Car.java

```diff
@@ -27,11 +27,19 @@ public int getDistance() {
         return distance;
     }
 
-    public boolean moveable(int value) {
-        return value >= MOVE_LIMIT;
+    public boolean moveable(RandomNo value) {
```

Comment:
```suggestion
    public boolean moveable(int value) {
```

종속성을 조금이나마 덜기 위해 
간편하게 `int`값으로 전달하는게 더 좋겠네요 😄